gravityflow/gravityflow
# Fix File Upload field editing for User Input Workflow Step

## Description
This PR fixes the issue where the editable File Upload field on the User Input Workflow Step was not displaying options for editing the File Upload field data.

This fixes [gravityflow/backlog#161](https://github.com/gravityflow/backlog/issues/161)

## Testing instructions
1. Create a Form with a File Upload field.
2. Create a User Input Workflow Step for the form with the File Upload field as editable.
3. Process an entry for the form, and upload a file on the File Upload filed (created on Step 1).
4. Go to Workflow Inbox (Admin Dashboard) and the User Input step for the entry created on step 3.
5. Confirm, the file upload field displays without the options to edit/remove the file.
6. Go to Workflow Inbox (Front-End) and the User Input step for the entry created on step 3.
7. Confirm, the file upload field displays fine
8. Now switch to this branch and repeat steps 4 to 7. This time confirm the icons appear fine on Admin Dashboard and Front-End.
9. Switch to Gravity Forms 2.5.6 and 2.4, and repeat step 8.

## Automated Test Enhancements
N/A
 
## Screenshots
N/A

## Documentation Changes
N/A

## Checklist:
